src/MemoryRouter.tsx

```diff
@@ -29,9 +33,9 @@ function getRouteAsPath(pathname: string, query: ParsedUrlQuery) {
 
 export type UrlObject = {
   pathname: UrlWithParsedQuery["pathname"];
+  path?: UrlWithParsedQuery["path"];
```

Comment:
I don't see this being used, did I miss it?

Reply:
Definitely vestigial from an initial path I went down.